### ISSUE TITLE
Error setting property 'fill' of RNSVGPath

### ISSUE THREAD

**Issue by sahilpandya** [react-native-community/react-native-svg#1330]
# Bug
Error setting property 'fill' of RNSVGPath with tag #5403: JSON value '4288387995' of type NSNumber cannot be converted to NSArray

#### Unexpected behavior
Svg is not working, when trying to see or open svg component, app is going to crash.

## Environment info
React-native-svg: ^12.0.3
React native info output:
  Binaries:
    Node: 10.16.3 - ~/.nvm/versions/node/v10.16.3/bin/node
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  IDEs:
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.12.0 => 16.12.0
    react-native: 0.61.5 => 0.61.5
  npmGlobalPackages:
    react-native-cli: 2.0.1

Library version: ^12.0.3

## Steps To Reproduce

1. npm i react-native-svg
2. cd ios
3. pod install
4. Open your app and go to SVG compoent.

Describe what you expected to happen:

1. Svg file should be working fine.

**Comment by msand** [comment]
Did you have an earlier version installed from before? Did you try clearing the metro cache?

**Comment by timwachsener** [comment]
Getting a similar error:

```
Error while updating property 'fill' of a view managed by: RNSVGPath

null

java.lang.Double cannot be cast to java.lang.String
```

**Comment by phsantiago** [comment]
Same problem here, fixed downgrading to `v9.13.6`

**Comment by Anapher** [comment]
It seems like the error came with version 11.0.1, as it is still working in 11.0.0.

**Comment by viniarruda** [comment]
> Same problem here, fixed downgrading to `v9.13.6`

It's work for me ! 🚀 🚀

**Comment by nerychucuy** [comment]
> Same problem here, fixed downgrading to `v9.13.6`

Worked for me too!

**Comment by sahilpandya** [comment]
Due to older version didn't unlink or removed properly so it was not working, After removing the  cache and removed everything then did a new install will works.
